Jarfile

```diff
@@ -1,12 +1,12 @@
-jar 'org.kill-bill.billing:killbill-api', '0.50'
-jar 'org.kill-bill.billing.plugin:killbill-plugin-api-currency', '0.23'
-jar 'org.kill-bill.billing.plugin:killbill-plugin-api-invoice', '0.23'
-jar 'org.kill-bill.billing.plugin:killbill-plugin-api-notification', '0.23'
-jar 'org.kill-bill.billing.plugin:killbill-plugin-api-payment', '0.23'
-jar 'org.kill-bill.billing.plugin:killbill-plugin-api-control', '0.23'
-jar 'org.kill-bill.billing.plugin:killbill-plugin-api-catalog', '0.23'
-jar 'org.kill-bill.billing.plugin:killbill-plugin-api-entitlement', '0.23'
-jar 'org.kill-bill.billing:killbill-util:tests', '0.18.0'
-jar 'org.kill-bill.billing:killbill-util', '0.18.0'
+jar 'org.kill-bill.billing:killbill-api', '0.51.27'
+jar 'org.kill-bill.billing.plugin:killbill-plugin-api-currency', '0.24.8'
+jar 'org.kill-bill.billing.plugin:killbill-plugin-api-invoice', '0.24.8'
+jar 'org.kill-bill.billing.plugin:killbill-plugin-api-notification', '0.24.8'
+jar 'org.kill-bill.billing.plugin:killbill-plugin-api-payment', '0.24.8'
+jar 'org.kill-bill.billing.plugin:killbill-plugin-api-control', '0.24.8'
+jar 'org.kill-bill.billing.plugin:killbill-plugin-api-catalog', '0.24.8'
+jar 'org.kill-bill.billing.plugin:killbill-plugin-api-entitlement', '0.24.8'
+jar 'org.kill-bill.billing:killbill-util:tests', '0.19.18'
+jar 'org.kill-bill.billing:killbill-util', '0.19.18'
```

Comment:
Nit: technically because i released `0.20.0` we could move those to `0.20.0`. In practice, i don't think it matters too much. @wwjbatista For future ruby plugins, if any, you could use `0.20.0`